sympy/solvers/inequalities.py

```diff
@@ -1007,3 +1009,176 @@ def reduce_inequalities(inequalities, symbols=[]):
 
     # restore original symbols and return
     return rv.xreplace({v: k for k, v in recast.items()})
+
+
+class UnboundedLinearProgrammingError(Exception):
+    def __init__(self, *args):
```

Comment:
I would not try to define custom methods for exceptions unless it is significantly more fancier.
`UnboundedLinearProgrammingError(Your error message)` should work even if you don't define `__init__, __str__`

Reply:
I get what you want, but I don't quite understand how it would work without the defining  __init__ and __str__ ?
Do you have similar code example I can look at?

I used template that I found online regarding custom exceptions..

Reply:
A custom exception doesn't need any methods. This will work fine if you just delete the `__init__` and `__str__` methods.

Reply:
ok, I deleted them.
Tell me if there is anything more that needs to be done :)